codegen/src/test/resources/software/amazon/awssdk/codegen/poet/paginators/customization.config

```diff
@@ -4,5 +4,11 @@
         "nestedContainers",
         "operationWithNoInputOrOutput"
     ],
-    "verifiedSimpleMethods" : ["paginatedOperationWithResultKey"]
+    "verifiedSimpleMethods" : ["paginatedOperationWithResultKey"],
+    "paginationCustomization": {
```

Comment:
Can we be more descriptive on naming for all of these values?

Is there ever a case where sync and async won't have the same feature set?

Reply:
>> Is there ever a case where sync and async won't have the same feature set?

No, I can use the same value for both. Initially i was going with the fqcn of the class spec file, then decided to keep it as the class name. I can update to use same value for both sync and async

>> Can we be more descriptive on naming for all of these values?

I put the details in the javadoc of the class spec files. Is the following name better?
`ReturnSameTokenGivenInRequestToIndicateLastPage`

Reply:
Was more concerned with just the sync / async keys. Having just the one key is nice now. Probably can go with a shorter name...

LastPageHasPreviousToken?

Fine with whatever really since it's just a customization value.